lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
I cannot imagine how unlucky should be visitor to receive collision with 32-character string. Especially if we cleanup them. Storing all possible values should take more that 3*10 ^ 29 Gb of data in database.